src/main/resources/application.yml

```diff
@@ -65,3 +65,7 @@ jwt:
                 odND6HPvvvmgbK1/2qTHn/HJjWUm11LUC73BszR32BKbdEEhxPQnnwswVekWzPi1
                 IwIDAQAB
                 -----END PUBLIC KEY-----}"
+
+rest:
```

Comment:
can you add also properties and RestTemplateConfiguration to handle timeout configuration?
https://github.com/pagopa/p4pa-pu-bff/blob/develop/src/main/java/it/gov/pagopa/pu/bff/config/RestTemplateConfig.java